### ISSUE TITLE
Add option to allow use of CPU adapters (like llvmpipe)

### ISSUE THREAD

**Issue by niansa** [doitsujin/dxvk#4121]
Currently, DXVK simply skips all CPU adapters. This is a good default, but it should be overridable.

Maybe this skip shouldn't occur if `DXVK_FILTER_DEVICE_NAME` is set.

**Comment by doitsujin** [comment]
This is already possible by setting `VK_ICD_FILENAMES` to only include CPU adapters, but yeah, allowing them when the device filter is set makes sense too.

**Comment by niansa** [comment]
Nope, says `Skipping CPU adapter: llvmpipe (LLVM 17.0.6, 256 bits)` for me.

**Comment by Blisto91** [comment]
Post the command you used

**Comment by doitsujin** [comment]
Works here so there's probably something wrong on your end.

**Comment by niansa** [comment]
`VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/lvp_icd.i686.json wine game32.exe`

**Comment by Blisto91** [comment]
I've tested it locally and it works fine here. Try with `VK_DRIVER_FILES` instead.

**Comment by niansa** [comment]
Nope, no difference :-/
Still says this:
```
warn:  Skipping CPU adapter: llvmpipe (LLVM 17.0.6, 256 bits)
info:  AMD Radeon RX 7900 XT (RADV NAVI31)
...
```

**Comment by niansa** [comment]
`bwrap --bind / / --ro-bind ~/Programs/OSS/mesa/inst/share/vulkan/icd.d /usr/share/vulkan/icd.d wine32 game32.exe`
